### PR TITLE
Indicator light for network connection

### PR DESCRIPTION
**Pull request by jessgusclark**
Uses the existing Bootstrap button styles, works in all three network environments, and tooltip lets you know what's up.

Also includes node-sass.

Needs translations

## screenshots: 
image

image

image

### CODE REVIEW

File: src/languages/en.json [hunk 1]
```diff
@@ -219,5 +219,7 @@
   "no_wallet": "No wallet detected",
   "network_mismatch": "Network Mismatch",
   "connect_to_network": "Connect your wallet to",
-  "current_connected": "Your wallet is currently connected to"
+  "current_connected": "Your wallet is currently connected to",
+  "connected_successful": "Connected Successful",
```
Comment (jessgusclark) [inline src/languages/en.json]: I am going to change this to `Connected Successfully`, but will wait to until after the translations come back.

File: src/app/auth/reducer.js [hunk 2]
```diff
@@ -77,6 +77,7 @@ export default (state = initialState, action) => {
       name: null,
       storageName: localStorage.getItem('name'),
       isOwner: false,
+      networkMatch: true,
```
Comment (jessgusclark) [inline src/app/auth/reducer.js]: My justification for setting this to true and not rechecking is that the only way to log out is to be logged in originally, which requires networkMatch set to true.

File: src/assets/css/sass/_header.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    height: 6px;
+    border-radius: 50%;
+    background: #40d512;
+  }
+}
+
+.indicator.btn-outline-danger {
+  span::before {
+    background-color:#dc3545;
+  }
```
Comment (ilanolkies) [inline src/assets/css/sass/_header.scss]:
```suggestion
  }
  
```

File: src/assets/css/sass/_header.scss [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+}
```
Comment (ilanolkies) [inline src/assets/css/sass/_header.scss]:
```suggestion
}

```

File: src/app/components/HeaderComponent.js [hunk 5]
```diff
@@ -55,6 +55,7 @@ const HeaderComponent = (props) => {
             <LanguageSelectContainer />
             <StartButton />
           </Form>
+          <IndicatorLight />
```
Comment (ilanolkies) [inline src/app/components/HeaderComponent.js]: I think it looks better beside rif logo, at the left part.

Comment (jessgusclark) [reply src/app/components/HeaderComponent.js]: I can move it beside the logo. I think to the right of the logo (second screenshot) looks better though.

### left of the logo

image

### right of the logo

image

Comment (ilanolkies) [reply src/app/components/HeaderComponent.js]: Right of the logo 😁

Comment (jessgusclark) [reply src/app/components/HeaderComponent.js]: Moved with [807623e](https://github.com/rnsdomains/rns-manager-react/pull/123/commits/807623e78c693e9e4e5ef78c6ef4ad32454ee565)

**Comment by ilanolkies** [comment]
Closes #115

**Pull request merged**